corehq/ex-submodules/casexml/apps/phone/xml.py

```diff
@@ -107,3 +107,8 @@ def get_data_element(name, dict):
         sub_el.attrib = {"key": k}
         elem.append(sub_el)
     return elem
+
+
+def synclog_id_from_restore_payload(restore_payload):
```

Comment:
whoops, yeah i thought i'd need it in the other code but then realized the streaming made it not possible.